prestodb/client.py

```diff
@@ -539,18 +529,25 @@ def execute(self):
 
         response = self._request.post(self._sql)
         status = self._request.process(response)
+        if status.next_uri is None:
+            self._finished = True
```

Comment:
nit . these lines seemed to be pulled up. If it is not needed let us keep the previous order.

Reply:
done